### PR TITLE
NOISSUE - Add Loki-Stack to charts

### PR DESCRIPTION
**Pull request by blokovi**

### CODE REVIEW

File: charts/mainflux/values.yaml [hunk 1]
```diff
@@ -330,3 +330,36 @@ nginx_internal:
...
+  fluent-bit:
+    enabled: false
+  grafana:
+    enabled: true
+    sidecar:
+      datasources:
+        enabled: true
+    image:
+      tag: 8.1.6
+    # grafana.ini:
```
Comment (manuio) [inline charts/mainflux/values.yaml:352]: Maybe you should uncomment?

Comment (blokovi) [reply charts/mainflux/values.yaml]: @manuio I uncomment it and put `example.com` for host

**Review by manuio — APPROVED** [review]
LGTM

**Review by drasko — APPROVED** [review]
LGTM

**Pull request merged**